src/test/components/ActiveTabTimeline.test.js

```diff
@@ -27,21 +22,25 @@ import { getFirstSelectedThreadIndex } from '../../selectors/url-state';
 import { changeSelectedThreads } from '../../actions/profile-view';
 import { ensureExists, getFirstItemFromSet } from '../../utils/flow';
 
+import { storeWithProfile } from '../fixtures/stores';
+import { getProfileWithNiceTracks } from '../fixtures/profiles/tracks';
+import { fireFullClick } from '../fixtures/utils';
+import { addActiveTabInformationToProfile } from '../fixtures/profiles/processed-profile';
+import { autoMockCanvasContext } from '../fixtures/mocks/canvas-context';
+import {
+  autoMockElementSize,
+  getElementWithFixedSize,
+} from '../fixtures/mocks/element-size';
+
 describe('ActiveTabTimeline', function() {
   autoMockCanvasContext();
+  autoMockElementSize({ width: 200, height: 300 });
   beforeEach(() => {
-    jest.spyOn(ReactDOM, 'findDOMNode').mockImplementation(() => {
-      // findDOMNode uses nominal typing instead of structural (null | Element | Text), so
-      // opt out of the type checker for this mock by returning `any`.
-      const mockEl = ({
-        getBoundingClientRect: () => getBoundingBox(300, 300),
-      }: any);
-      return mockEl;
-    });
-
     jest
-      .spyOn(HTMLElement.prototype, 'getBoundingClientRect')
-      .mockImplementation(() => getBoundingBox(200, 300));
+      .spyOn(ReactDOM, 'findDOMNode')
+      .mockImplementation(() =>
+        getElementWithFixedSize({ width: 300, height: 300 })
```

Comment:
note: I'm not sure why we had different values for the mock for `findDOMNode` and the mock for `getBoundingClientRect` (was that on purpose?), but I decided to keep them so that snapshots wouldn't change.

Reply:
Hmm, I'm not sure. I think I wrote this part and it's likely to be a copy paste from another test. It looks like a bug indeed.

Edit: yeah, it's coming from here:
https://github.com/firefox-devtools/profiler/blob/81120695dbc6b7be9dbd93c4ff99218d6c3ff02b/src/test/components/Timeline.test.js#L405-L417

Can't think of an idea why it would need to be different. Probably a bug. But it's not this PRs work anyway. It's good to keep it for now.

Reply:
I was thinking that maybe one is for the container and the other for something inside the container... but didn't want to dig more :D